tools/deb/istio-iptables.sh

```diff
@@ -46,6 +46,8 @@ function usage() {
   echo '  -x: Comma separated list of IP ranges in CIDR form to be excluded from redirection. Only applies when all '
   # shellcheck disable=SC2016
   echo '      outbound traffic (i.e. "*") is being redirected (default to $ISTIO_SERVICE_EXCLUDE_CIDR).'
+  echo '  -r: Comma separated list of internal interfaces to be included for redirection. (optional)'
```

Comment:
Comma separated list of virtual interfaces whose inbound traffic (from VM) will be treated as outbound